### DISCUSSION TITLE
Is the project dead?

### DISCUSSION THREAD

**Discussion by karolswdev** [yuin/gopher-lua#506] [General]
Hey guys. I'm creating an application in `golang` and am strongly considering `gopher-lua`. I noticed that the last merge into `main` was ~10 months ago. Is there a roadmap for further development of this project? Are there other go projects I should be looking to fulfill my need in case this project has stopped being supported?

**Comment by rusenask** [comment]
just use it, there's nothing wrong/missing :)
